### PR TITLE
EDU-3012: Update Failure and Call Stack details to current Web UI

### PR DESCRIPTION
**Pull request by fairlydurable**
Replace outdated terms and screenshot with current representation

### CODE REVIEW

File: docs/getting_started/go/first_program_in_go/index.md [hunk 1]
```diff
@@ -607,9 +607,8 @@ You can view more information about the process in the [Temporal Web UI](localho
 
 ![The next Activity](images/activity_failure.png)
 
-Click the **Stack Trace** link to see a stack trace showing you the errors, as well as details about the pending Activity:
-
-![The stack trace of the Activity](images/stack_trace.png)
+Click any ActivityTaskFailed event link to see Failure detail information and a Call Stack trace.
```
Comment (tomwheeler) [inline docs/getting_started/go/first_program_in_go/index.md:610]:
```suggestion
Click any `ActivityTaskFailed` event link to see the Failure details and call stack.
```

NOTE: My suggested wording better matches what is in the current UI, although the screenshot used in the tutorial is outdated.

File: docs/getting_started/python/first_program_in_python/index.md [hunk 2]
```diff
@@ -586,9 +586,8 @@ You can view more information about the process in the [Temporal Web UI](http://
 
 ![The next Activity](images/activity_failure.png)
 
-5. Click the **Stack Trace** link to see a stack trace showing you the errors, as well as details about the pending Activity:
-
-![The stack trace of the Activity](images/stack_trace.png)
+5. Click any ActivityTaskFailed event link to see Failure detail information and a Call Stack trace.
```
Comment (tomwheeler) [inline docs/getting_started/python/first_program_in_python/index.md:589]:
```suggestion
5. Click any `ActivityTaskFailed` event link to see the Failure details and call stack.
```

File: docs/getting_started/typescript/first_program_in_typescript/index.md [hunk 3]
```diff
@@ -600,9 +600,8 @@ You can view more information about the process in the [Temporal Web UI](localho
 
 ![The next Activity](images/activity_failure.png)
 
-Click the **Stack Trace** link to see a stack trace showing you the errors, as well as details about the pending Activity:
-
-![The stack trace of the Activity](images/stack_trace.png)
+Click any ActivityTaskFailed event link to see Failure detail information and a Call Stack trace.
```
Comment (tomwheeler) [inline docs/getting_started/typescript/first_program_in_typescript/index.md:603]:
```suggestion
Click any `ActivityTaskFailed` event link to see the Failure details and call stack.
```

**Review by tomwheeler — CHANGES_REQUESTED** [review]
Suggested putting the Event name in monospace, as per our style, and updating the wording to reflect the current UI.

File: docs/getting_started/go/first_program_in_go/index.md (hunk 1 above)
Comment (fairlydurable) [reply docs/getting_started/go/first_program_in_go/index.md]: Thank you. Adopted.

File: docs/getting_started/python/first_program_in_python/index.md (hunk 2 above)
Comment (fairlydurable) [reply docs/getting_started/python/first_program_in_python/index.md]: Thank you. Adopted.

File: docs/getting_started/go/first_program_in_go/index.md (hunk 1 above)
Comment (fairlydurable) [reply docs/getting_started/go/first_program_in_go/index.md]: Thank you, adopted.

File: docs/getting_started/typescript/first_program_in_typescript/index.md (hunk 3 above)
Comment (fairlydurable) [reply docs/getting_started/typescript/first_program_in_typescript/index.md:603]: Thank you adopted.

**Review by tomwheeler — APPROVED** [review]
Ship it!

**Pull request merged**